Translations/hr.po

```diff
@@ -4085,333 +3903,297 @@ msgstr ""
 
 #: Source/items.cpp:1961
-#, fuzzy
 msgid "increase vitality"
-msgstr "Vitalnost:"
+msgstr "povećava vitalnost"
 
 #: Source/items.cpp:1966
-#, fuzzy
 msgid "decrease strength"
-msgstr "Snaga:"
+msgstr "smanjuje snagu"
 
 #: Source/items.cpp:1970
-#, fuzzy
 msgid "decrease dexterity"
-msgstr "Vještina:"
+msgstr "smanjuje vještinu"
 
 #: Source/items.cpp:1974
-#, fuzzy
 msgid "decrease vitality"
-msgstr "Vitalnost:"
+msgstr "smanjuje vitalnost"
 
 #: Source/items.cpp:1978
 msgid "restore some life and mana"
-msgstr ""
+msgstr "djelomično obnavlja život i manu"
 
 #: Source/items.cpp:1982
 msgid "restore all life and mana"
-msgstr ""
+msgstr "potpuno obnavlja život i manu"
 
 #: Source/items.cpp:1997 Source/items.cpp:2022
-#, fuzzy
 msgid "Right-click to read"
-msgstr "Greška, direktorij ima samo dozvolu čitanja"
+msgstr "Desno klikni za čitanje"
 
 #: Source/items.cpp:2001
-#, fuzzy
 msgid "Right-click to read, then"
-msgstr "Greška, direktorij ima samo dozvolu čitanja"
+msgstr "Desno klikni za čitanje, zatim"
 
 #: Source/items.cpp:2003
 msgid "left-click to target"
-msgstr ""
+msgstr "lijevo klikni na metu"
 
 #: Source/items.cpp:2008
 msgid "Right-click to use"
-msgstr ""
+msgstr "Desno klikni za korištenje"
 
 #: Source/items.cpp:2013 Source/items.cpp:2018
 msgid "Right click to use"
-msgstr ""
+msgstr "Desno klikni za korištenje"
 
 #: Source/items.cpp:2026
-#, fuzzy
 msgid "Right click to read"
-msgstr "Greška, direktorij ima samo dozvolu čitanja"
+msgstr "Desno klikni za čitanje"
 
 #: Source/items.cpp:2030
 msgid "Right-click to view"
-msgstr ""
+msgstr "Desno klikni za pogledati"
 
 #: Source/items.cpp:2038
 msgid "Doubles gold capacity"
-msgstr ""
+msgstr "Udvostručuje kapacitet zlata"
 
 #: Source/items.cpp:2050 Source/stores.cpp:281
-#, fuzzy
-#| msgid "Requirements:"
 msgid "Required:"
-msgstr "Zahtjevi:"
+msgstr "Potrebno:"
 
 #: Source/items.cpp:2052 Source/stores.cpp:283
-#, fuzzy
 msgid " {:d} Str"
-msgstr "Podržano igrača: {:d}"
+msgstr " {:d} Snage"
 
 #: Source/items.cpp:2054 Source/stores.cpp:285
-#, fuzzy
 msgid " {:d} Mag"
-msgstr "Podržano igrača: {:d}"
+msgstr " {:d} Magije"
 
 #: Source/items.cpp:2056 Source/stores.cpp:287
-#, fuzzy
 msgid " {:d} Dex"
-msgstr "Podržano igrača: {:d}"
+msgstr " {:d} Vještine"
 
 #. TRANSLATORS: {:s} will be a Character Name
 #: Source/items.cpp:3431 Source/player.cpp:3001
-#, fuzzy
 msgid "Ear of {:s}"
-msgstr ""
-"Nemoguće zapisivanje lokacije:\n"
-"{:s}"
+msgstr "Uho od {:s}"
 
 #: Source/items.cpp:3732
-#, fuzzy
 msgid "chance to hit: {:+d}%"
-msgstr "Podržano igrača: {:d}"
+msgstr "šanse za pogodak: {:+d}%"
 
 #: Source/items.cpp:3735
-#, fuzzy, no-c-format
+#, no-c-format
 msgid "{:+d}% damage"
-msgstr "Podržano igrača: {:d}"
+msgstr "{:+d}% oštećenje"
 
 #: Source/items.cpp:3738 Source/items.cpp:3936
-#, fuzzy
 msgid "to hit: {:+d}%, {:+d}% damage"
-msgstr ""
-"{:s}\n"
-"\n"
-"Greška se dogodila na: {:s} redak {:d}"
+msgstr "pogodak: {:+d}%, {:+d}% oštećenje"
```

Comment:
From player feedback it sounds like this might be more appropriate for "damaged" and not "damage", would "šteta" be better?